client/src/lib/Router.js

```diff
@@ -27,8 +27,9 @@ function resolveURLToBase(path) {
     return absolutePath;
   }
 
-  // Remove base url from absolute path, save for trailing `/` which Page.js requires
-  return absolutePath.substring(absoluteBase.length - 1);
+  // Remove base url from absolute path, save for leading `/` which Page.js requires
+  const regex = new RegExp(`^${absoluteBase}/?`);
```

Comment:
Should regex escape asbsoluteBase e.g. `.` should become `\.`

Though why not just use the substring above? then ltrim any '/' and then prefix '/' back in

Reply:
`rtrim` and `suffix` - it's meant to be on the end. But regardless, there's no simple `rtrim` function in javascript.

And even if there was, this doesn't just "remove and add a slash", it's removing the `absoluteBase` from the `absolutePath`. e.g. if `absoluteBase` is `https://www.example.com/` and `absolutePath` is `https://www.example.com/some-path/`, the result of this operation is `/some-path/`.

You're correct that the variable needs to be escaped, though. I'll use lodash for that.

Reply:
Done